fxa-oauth-server/lib/config.js

```diff
@@ -33,7 +33,7 @@ const conf = convict({
     doc: 'Comma-separated list of secret keys for verifying server-to-server JWTs',
     env: 'AUTH_SERVER_SECRETS',
     format: Array,
-    default: []
+    default: ['megaz0rd']
```

Comment:
I deliberately didn't do this in the original PR because I didn't want to risk shipping a default secret to production. What issue were you seeing that caused you to add it, and can we fix it by adding this in e.g. a dev config rather than the defaults?

Reply:
Ok! Let's add it to dev